Checkout/Sample/Source/Card Tokenisation/Card Details/CardDetailsTableViewCell.swift

```diff
@@ -7,30 +7,129 @@
 
 import UIKit
 
+protocol CardDetailsTableViewCellDelegate: AnyObject {
+  var cardNumberDelegate: UITextFieldDelegate { get }
+}
+
 class CardDetailsTableViewCell: UITableViewCell {
   @IBOutlet weak var cardNumberTextField: UITextField!
   @IBOutlet weak var expiryMonthTextField: UITextField!
   @IBOutlet weak var expiryYearTextField: UITextField!
   @IBOutlet weak var cvvTextField: UITextField!
+  @IBOutlet weak var expiryTextFieldStackView: UIStackView!
+  @IBOutlet weak var expiryDatePickerField: UITextField!
+
+  struct ViewModel {
+    var onCardNumberChange: ((String?) -> Void)?
+    var onCVVChange: ((String?) -> Void)?
+
+    var onExpiryMonthStringChange: ((String?) -> Void)?
+    var onExpiryYearStringChange: ((String?) -> Void)?
+
+    var onExpiryIntChange: ((_ month: Int, _ year: Int) -> Void)?
```

Comment:
Is this really a ViewModel? Being a collection of callbacks it feels like it wants to be a delegate. Also, same thing about hidden retain cycles applies again.